internal/cli/atlas/serverless/backup/restores/create.go

```diff
@@ -144,7 +144,7 @@ func CreateBuilder() *cobra.Command {
 		Short: "Start a restore job for your serverless instance.",
 		Long: `If you create an automated or pointInTime restore job, Atlas removes all existing data on the target cluster prior to the restore.
 
-` + fmt.Sprintf(usage.RequiredRole, "Project Owner"),
+` + fmt.Sprintf("%s\n%s", fmt.Sprintf(usage.RequiredRole, "Project Owner"), "This command is supported only for M10+ clusters."),
```

Comment:
we should use active voice instead of passive voice
```suggestion
` + fmt.Sprintf("%s\n%s", fmt.Sprintf(usage.RequiredRole, "Project Owner"), "Atlas supports this command only for M10+ clusters."),
```